utils/calico-uts.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+
+MAINTAINER Alex Chan <alexchan@projectcalico.org>
+
+RUN apt-get update
+RUN echo "APT::GET::Assume-Yes \"true\";" >> /etc/apt/apt.conf
+
+# tox installs python-posix-spawn and python-etcd from Git
+RUN apt-get install git
```

Comment:
You could move them to a setup script and run that as one step.  That'd also make it easier to create another container based on CentOS, for example.